modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/integration/DataTypesTest.java

```diff
@@ -474,6 +446,9 @@ public void testNumericConversion() {
     /** */
     @Test
     public void testFunctionArgsToNumericImplicitConversion() {
+        if (sqlTxMode != SqlTransactionMode.NONE)
```

Comment:
assertQuery has check that tx exists in certain modes.
It required to check that test logic correct and starts tx if required.
I skip two tests in `DataTypeTest` with `assumeTrue`, because them check only queries without any actual data select.